src/aiidalab_qe/workflows/__init__.py

```diff
@@ -73,7 +73,7 @@ def define(cls, spec):
             spec.expose_inputs(
                 plugin_workchain,
                 namespace=name,
-                exclude=entry_point["exclude"],
+                exclude=entry_point.get("exclude", []),
```

Comment:
```suggestion
                exclude=entry_point.get("exclude"),
```
I think the default value of exclude is `None`, no? It is not good to use mutable values in a function constructor.

Reply:
I checked the sources:
```
exclude: Optional[Sequence[[str]]] = None
```
The default one is `None`, but passing `[]` as a variable (note, here is not a constructor) is also safe. I used `None` as you suggested.